### PR TITLE
Switch Travis to GitHub Actions

### PR DESCRIPTION
**Pull request by frbuceta**
See strongloop/loopback-next#7054

## Checklist

- [x] DCO (Developer Certificate of Origin) [signed in all commits](https://loopback.io/doc/en/contrib/code-contrib.html)
- [x] `npm test` passes on your machine
- New tests added or existing tests modified to cover all changes
- Code conforms with the [style guide](https://loopback.io/doc/en/contrib/style-guide-es6.html)
- [x] Commit messages are following our [guidelines](https://loopback.io/doc/en/contrib/git-commit-messages.html)

### CODE REVIEW

**Comment by frbuceta** [comment]
Thats weird! Actions are not executed. Could they be disabled for me? @strongloop/loopback-maintainers

**Comment by achrinza** [comment]
IIRC, the incoming PR must be from a branch in the main repo (i.e. not a fork); Otherwise, GitHub will not execute the changes until it's merged.

**Comment by frbuceta** [comment]
> IIRC, the incoming PR must be from a branch in the main repo (i.e. not a fork); Otherwise, GitHub will not execute the changes until it's merged.

I don't have permissions to create a branch in this repository, can you give me access?

**Comment by mrmodise** [comment]
@frbuceta do a pull request to your own folked master it will run pipeline and reference here. @bajtos will need to update the repo settings for it to run Github actions

**Comment by frbuceta** [comment]
This is the proof that it works correctly https://github.com/frbuceta/loopback-connector-postgresql/pull/1#issue-565051760

**Comment by dhmlau** [comment]
@frbuceta, thanks for the PR.
I checked the settings. The `Actions permission` is set to "allow all actions". Not sure where else needs to be changed.
@bajtos, any idea?

**Comment by bajtos** [comment]
I suspect it's the cause described by @achrinza in his comment https://github.com/strongloop/loopback-connector-postgresql/pull/473#issuecomment-770512412 :

> IIRC, the incoming PR must be from a branch in the main repo (i.e. not a fork); Otherwise, GitHub will not execute the changes until it's merged.

I think the next step is to merge this PR and then fix any issues in follow-up PRs.

File: .github/workflows/continuous-integration.yml
Comment (bajtos) [inline .github/workflows/continuous-integration.yml:1]: Let's use the same name as we have in loopback-next please, for consistency.

See https://github.com/strongloop/loopback-next/blob/8077f8e5954e9d0b638808a1efb8b703f9061b5f/.github/workflows/continous-integration.yaml.

```suggestion
name: Continous Integration
```

**Comment by frbuceta** [comment]
> I suspect it's the cause described by @achrinza in his comment [#473 (comment)](https://github.com/strongloop/loopback-connector-postgresql/pull/473#issuecomment-770512412) :
> 
> > IIRC, the incoming PR must be from a branch in the main repo (i.e. not a fork); Otherwise, GitHub will not execute the changes until it's merged.
> 
> I think the next step is to merge this PR and then fix any issues in follow-up PRs.

As it is now, it should work. You can merge

**Review by bajtos — CHANGES_REQUESTED** [review]
The proposal looks great to me!

Let's use a different workflow name please per my comment below.

File: .github/workflows/continuous-integration.yml
Comment (frbuceta) [reply .github/workflows/continuous-integration.yml]: Done

**Review by bajtos — APPROVED** [review]

**Comment by bajtos** [comment]
Landed, thank you @frbuceta ❤️

**Comment by bajtos** [comment]
Hmm, the first build failed - see https://github.com/strongloop/loopback-connector-postgresql/actions/runs/530701696

**Comment by bajtos** [comment]
Reverting this change in https://github.com/strongloop/loopback-connector-postgresql/pull/474.

Let's try the new setup in your personal fork first, @frbuceta. Create a feature branch, open a PR against your own repo. Once you fix all issues, then the next step is open a PR from your feature branch to the upstream repo (https://github.com/strongloop/loopback-connector-postgresql).

**Comment by agnes512** [comment]
this might help https://github.com/marketplace/actions/setup-postgresql

**Comment by frbuceta** [comment]
> this might help https://github.com/marketplace/actions/setup-postgresql

This is a shortcut to what we are using right now in the code. This afternoon we have executed (@bajtos) the code in my fork but here it is not working correctly. @bajtos is investigating why this happens.

See https://github.com/frbuceta/loopback-connector-postgresql/pull/1

**Comment by bajtos** [comment]
Sorry for the delay. There is a lot on my plate these days and what's worse, I fell sick this week. It makes me wonder, would you @agnes512 have bandwith to take over this work from me? My idea was to create a new feature branch in this repo, cherry-pick @frbuceta's commit, open a pull request and then tweak the settings or our tests until all GH actions are passing.

**Pull request merged**
